lib/hydra/derivatives/services/capability_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'open3'
+
+module Hydra::Derivatives
+  class CapabilityService
+    attr_accessor :ffmpeg_output
+    def capture_output
+      @ffmpeg_output = Open3.capture3('ffmpeg -codecs').to_s
+    rescue
+      Logger.warn('Unable to find ffmpeg')
+      @ffmpeg_output = ""
+    end
+
+    def fdk_aac?
```

Comment:
Calling this method assumes that `capture_output` has been called (or at least been set via the `attr_accessor :ffmpeg_output`). I'm looking at the assumed temporal coupling of these methods.